packages/react-vanilla-components/src/components/ResponsiveGrid.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useCallback, useContext } from 'react';
+import { FormContext, getRenderer } from '@aemforms/af-react-renderer';
+import { PROPS_PANEL } from '../utils/type';
+
+
+const ResponsiveGrid = (props: PROPS_PANEL) => {
+    //   @ts-ignore
+    const context = useContext(FormContext);
+
+    const getItemClassNames = (appliedCssClassNames: string, index: number, fieldType: string) => {
+        const name = props?.[':itemsOrder']?.[index] || '';
+        const responsiveClassName = props?.columnClassNames?.[name];
+        return responsiveClassName ? `${name} ${fieldType === 'panel' ? 'responsivegrid' : 'base'} ${appliedCssClassNames || ''} ${responsiveClassName}` : '';
+    };
+
+    const getChild = useCallback((child: any, index: number) => {
+        const Comp = getRenderer(child, context.mappings);
+        return Comp ? <Comp key={`${child.id}_${index}`} {...child} /> : null;
+    }, []);
+
+    return (
+        <>
+            {props?.items.map((el: any, index: number) => (
```

Comment:
Change the name 'el' to 'item,' and specify that the type of 'item' can be either 'panel' or 'field.